packages/tftp/tftp.0.1.4/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "1.2"
+maintainer: "mort@cantab.net"
+authors: "Richard Mortier <mort@cantab.net>"
+homepage: "https://github.com/mor1/ocaml-tftp"
+bug-reports: "https://github.com/mor1/ocaml-tftp/issues"
+license: "ISC"
+dev-repo: "https://github.com/mor1/ocaml-tftp.git"
+build: [
+  ["./configure" "--prefix=%{prefix}%" "--%{alcotest:enable}%-tests"]
+  [make "build"]
+  ["cp" "./tftp.opam/install" "./tftp.install"]
+]
+install: [
+  [make "install"]
+  [make "tftpd" "FS=direct" "NET=socket" "MIRFLAGS=--no-depext --no-opam"]
+]
+build-test: [make "test"]
+remove: ["ocamlfind" "remove" "tftp"]
+depends: [
+  "ocamlfind" {build}
+  "alcotest"
```

Comment:
sure; test-only in fact (though there aren't any yet). what's the syntax for that?